src/worker.py

```
@@ -2500,6 +2510,113 @@ async def api_express_activity_interest(activity_ref: str, req, env):
     }, "Interest recorded")
 
 
+async def api_generate_certificate(request, env, target_id):
```

Ruff (0.16.0)

[warning] Too many return statements (8 > 6)

(PLR0911)